Purchases.xcodeproj/project.pbxproj

```diff
@@ -728,7 +729,7 @@
 			isa = PBXGroup;
 			children = (
 				F591492426B994A100D32E58 /* StoreKitExtensions */,
-				35F82BBB26A9BFA60051DF03 /* Misc */,
+				35F82BBB26A9BFA60051DF03 /* FoundationExtensions */,
```

Comment:
just move it from the `Misc` folder to the existing `FoundationExtensions` folder